docs/secrets.md

```diff
@@ -2,7 +2,7 @@
 
 To publish to [Sonatype/Maven Central](https://central.sonatype.org/) you must obtain and install the following secrets on your repository:
 
-- `SONATYPE_USERNAME` and `SONATYPE_PASSWORD`
+- `SONATYPE_USERNAME` and `SONATYPE_PASSWORD` not your Sonatype/jira login credentials! The following section provides instructions for obtaining these.
```

Comment:
Hm, actually they _can_ be your sonatype credentials. Were they not working for you?

Reply:
No, and trying to use them seemed to result in github deleting all of the secrets in the project.

The docs definitely refer to two different processes for creating two distinct sets of credentials.

Reply:
> No, and trying to use them seemed to result in github deleting all of the secrets in the project.

That doesn't sound right. I would encourage you to open an issue with GitHub :)

Reply:
Sometimes helpful hearts cook up irritating ideas so nixing this PR will not hurt my feelings.

However, I have a lot more projects to publish, so if you're patient, I can experiment further and report back here.

Reply:
You're right.  this time the other set of credentials worked.  Please reject this pr.